bigbluebutton-html5/imports/ui/components/audio/local-echo/service.js

```diff
@@ -27,8 +27,14 @@ const playEchoStream = async (stream, loopbackAgent = null) => {
         loopbackAgent.stop();
       }
     }
-
-    audioElement.srcObject = streamToPlay;
+    const audioCtx = new AudioContext();
+    const source = audioCtx.createMediaStreamSource(stream);
```

Comment:
This bypasses the local loopback used in Chromium based browsers to account for their WebAudio bugs (eg [AEC, GC et al don't work unless they go through remote peers](https://bugs.chromium.org/p/chromium/issues/detail?id=687574)).
See our `useRtcLoopbackInChromium` flag. It helps reducing feedback loops - although the delay itself should help with that, I'd rather have the current behavior maintained at this point of the release (and for Chromium to be consistent with WebKit/Gecko in that regard).